public/locales/es-ES/grafana.json

```diff
@@ -46,28 +46,23 @@
     },
     "counts": {
       "alertRule_one": "",
-      "alertRule_many": "",
```

Comment:
Pulling in comment from the previous PR:

Ahh. The reason for this is that the English catalogue is the main reference. Any strings that exist in other languages but don't exist in English will be deleted.

As English doesn't have the __many plural, Crowdin will delete that translation key from all the other locales. It's is... problematic and will need to investigate the proper solution for it.

Semi related, because we don't have English __many plurals, this means that they're not able to be translated in Crowdin. The solution to both of these problems might be the same.


https://github.com/grafana/grafana/pull/76243#discussion_r1352766793